cupy/cuda/nvtx.pxd

```diff
@@ -0,0 +1,53 @@
+"""
+Wrapper for NVIDIA Tools Extension Library (NVTX)
+
+"""
+cdef extern from "cupy_cuda.h":
+    cdef int NVTX_VERSION
+    cdef int NVTX_EVENT_ATTRIB_STRUCT_SIZE
+    cdef int NVTX_NO_PUSH_POP_TRACKING
+    cdef enum nvtxColorType_t:
+        NVTX_COLOR_UNKNOWN
+        NVTX_COLOR_ARGB
+    cdef enum nvtxPayloadType_t:
+        NVTX_PAYLOAD_UNKNOWN
+        NVTX_PAYLOAD_TYPE_UNSIGNED_INT64
+        NVTX_PAYLOAD_TYPE_INT64
+        NVTX_PAYLOAD_TYPE_DOUBLE
+    cdef enum nvtxMessageType_t:
+        NVTX_MESSAGE_UNKNOWN
+        NVTX_MESSAGE_TYPE_ASCII
+        NVTX_MESSAGE_TYPE_UNICODE
+    ctypedef   signed char         int8_t
+    ctypedef   signed short       int16_t
+    ctypedef   signed int         int32_t
+    ctypedef   signed long long   int64_t
+    ctypedef unsigned char        uint8_t
+    ctypedef unsigned short      uint16_t
+    ctypedef unsigned int        uint32_t
+    ctypedef unsigned long long  uint64_t
+    ctypedef          void        wchar_t
+    cdef union payload_t:
+        uint64_t  ullValue
+        int64_t    llValue
+        double      dValue
+    cdef union message_t:
+        char*     ascii
+        wchar_t*  unicode
+    cdef struct nvtxEventAttributes_v1:
```

Comment:
`nvtxEventAttributes_v1` is almost same `nvtxEventAttributes_v2`.
Only `payload_t` is different.
```python
union payload_t
{
    uint64_t ullValue;
    int64_t llValue;
    double dValue;
    /* NVTX_VERSION_2 */
    uint32_t uiValue;
    int32_t iValue;
    float fValue;
} payload;
```

`cupy.cuda,nvtx,Mark` function do not use `payload`.
I think it is possible to use the same struct in version 1 and 2.